apps/crn-frontend/src/analytics/engagement/state.ts

```diff
@@ -81,6 +83,15 @@ export const useAnalyticsEngagement = (
   const [engagement, setEngagement] = useRecoilState(
     analyticsEngagementState(options),
   );
+
+  const resetEngagement = useResetRecoilState(
```

Comment:
We can use the objectID field instead of id since those are unique. That way we wouldn't need to force refetch. Example from user productivity
<img width="665" alt="Screenshot 2024-08-29 at 12 00 52" src="https://github.com/user-attachments/assets/bd31885f-4043-407a-bad5-0f3c5e05e039">

Reply:
nice, I'll do that! Thanks!